src/runtime-rs/crates/runtimes/src/shim_mgmt/client.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#![allow(dead_code)]
+
+// Defines the general client functions used by other components acting like
+// clients. To be specific, a client first connect to the socket, then send
+// request to destined URL, and finally handle the request(or not)
+
+use std::{path::Path, path::PathBuf, time::Duration};
+
+use super::server::mgmt_socket_addr;
+use anyhow::{anyhow, Context, Result};
+use hyper::{Body, Client, Method, Request, Response};
+use hyperlocal::{UnixClientExt, UnixConnector, Uri};
+
+/// Shim management client with timeout
+pub struct MgmtClient {
+    /// The socket *file path* on host file system
+    s_path: PathBuf,
```

Comment:
Can you use a longer and meaningful name instead of `s_path`, for example `socket_path`?

Reply:
Fixed